lib-injection/host_inject.rb

```diff
@@ -0,0 +1,113 @@
+return if ENV["DD_TRACE_SKIP_LIB_INJECTION"] == "true"
+require "rubygems"
+
+# This would return MAJOR.MINOR without PATCH version, for example 3.1, 3.2
+ruby_version = RUBY_VERSION.split(".")[0..1].join(".")
```

Comment:
```suggestion
ruby_version = RbConfig::CONFIG["ruby_version"]
```

I recommend using the same logic that bundler uses to resolve this partial ruby version: https://github.com/rubygems/rubygems/blob/be63f08d78b0fd9cf85cd8aa68aab9b850177069/lib/rubygems/defaults.rb#L38